modules/release-notes/pages/relnotes.adoc

```diff
@@ -6,6 +6,128 @@
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
 
+[#release-705]
+== Release 7.0.5 (December 2023)
```

Comment:
Incorrect year. This is a typo from my original PR #3020 

```suggestion
== Release 7.0.5 (December 2022)
```

Reply:
Ha, I changed the wrong line the wrong way   :facepalm: